test/node/miscellaneous.js

```diff
@@ -34,4 +34,9 @@ describe('Miscellaneous Tests', () => {
         const res = await client.getTelephonyData();
         assert(res);
     });
+
+    it('should get past conversations', async () => {
```

Comment:
why 10? since you don't check the number (and may not have 10) we should not pass that parameter. 
We should create tests that test each of the parameters of getConversations, but create a new pull request for that. I'd like to merge this one once the comments are addressed

Reply:
Should I remove this from this patch then?

Reply:
yes, lets put all conversation retrieval into a separate test suite